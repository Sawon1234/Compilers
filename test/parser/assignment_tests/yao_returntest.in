fun foo(y : Integer) : Integer = y * 2;

return 0;