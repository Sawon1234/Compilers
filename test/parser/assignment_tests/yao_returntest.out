fun foo < > ( y : Integer < > ) : Integer < > return y . times < > ( 2 ) ; return 0 ;